<launch>
	<arg name="namespace" default="yk_builder" />

	<!-- Launch outcomes.launch from vibro_tactile_toolbox package -->
	<include
		file="$(find vibro_tactile_toolbox)/launch/nist_outcomes.launch">
		<arg name="namespace" value="$(arg namespace)"/>
	</include>

	<!-- Launch terminators.launch from vibro_tactile_toolbox package -->
	<include
		file="$(find vibro_tactile_toolbox)/launch/terminators.launch">
		<arg name="namespace" value="$(arg namespace)"/>
	</include>

	<group ns="$(arg namespace)">
		<!-- Start pose_stamped_publisher node -->
		<node name="pose_stamped_publisher_node" pkg="vibro_tactile_toolbox" type="pose_stamped_publisher.py" output="screen">
			<param name="frequency" value="100" />
		</node>

		<!-- Start audio_outcome_republisher node -->
		<node name="audio_outcome_republisher_node" pkg="vibro_tactile_toolbox" type="audio_outcome_republisher.py" output="screen">
			<param name="frequency" value="10" />
			<param name="time_delay" value="5" />
		</node>
	</group>
</launch>
